marlowe-tutorial/doc/escrow-ex.adoc

```diff
@@ -0,0 +1,177 @@
+[#escrow-ex]
+== A first example
+
+This tutorial introduces a simple financial contract in pseudocode,
+before explaining how it is modified to work in Marlowe, giving the
+first example of a Marlowe contract.
+
+=== A simple escrow contract
+
+image:escrow.png[Escrow]
+
+Suppose that `+alice+` wants to buy a cat from `+bob+`, but neither of
```

Comment:
You don't need the `+` signs everywhere. Up to you if you want to blanket use them I guess.

Reply:
don't care, I'm not changing the content except to make it look like the plutus tutorial